index.js

```diff
@@ -1,11 +1,34 @@
 var fs = require('fs-extra')
+var minimatch = require('minimatch')
+
+function performMatch (entry, globs) {
+  for (var i = 0; i < globs.length; i++) {
+    if (minimatch(entry, globs[i])) {
+      return true
+    }
+  }
+}
 
 exports.readSync = readSync
-function readSync (dir) {
+function readSync (dir, options) {
+  options = options || {}
+  var includeFiles = options.includeFiles
+  var excludeFiles = options.excludeFiles || []
+
   var obj = {}
   var entries = fs.readdirSync(dir).sort()
   for (var i = 0; i < entries.length; i++) {
     var entry = entries[i]
+
+    if (includeFiles) {
+      if (!performMatch(entry, includeFiles)) {
```

Comment:
Entry here is just going to be the filename, not the path portion. I would expect to be able to match things based on the full path (not just the filename)...

Reply:
Good call

Reply:
Fixed the bug, and altered the tests to catch it